packages/client/src/webAPI/fullscreen.ts

```diff
@@ -0,0 +1,13 @@
+export function fullscreenAPI() {document.addEventListener("keydown", (e) => {
+  if (e.key === "f") {
+    toggleFullScreen();
+  }
+}, false);}
```

Comment:
запусти линтер пжл, добавьте хук на коммит что-ли, чтобы запустился перед коммитом.
```suggestion
export function fullscreenAPI() {
  document.addEventListener("keydown", (e) => {
    if (e.key === "f") {
      toggleFullScreen();
    }
  }, false);
}
```